HtmlControl.cs

```diff
@@ -411,7 +416,14 @@ protected override void OnPointerPressed(PointerPressedEventArgs e)
         {
             base.OnPointerPressed(e);
             LeftMouseButton = true;
-            _htmlContainer?.HandleLeftMouseDown(this, e);
+            if (e.ClickCount == 1)
+            {
+                _htmlContainer?.HandleLeftMouseDown(this, e);
+            }
+            else if (e.ClickCount == 2)
```

Comment:
I think it's better to use DoubleTapped event instead, so it can be handled with touch better

Reply:
HandleMouseDoubleClick expects PointerEventArgs, which we don't get on DoubleTapped, so I'm not sure how to pass it this data.

Reply:
It needs only current position
https://github.com/AvaloniaUI/Avalonia.HtmlRenderer/blob/b3997448d3f57823eff6f2190a80643bdfc5102d/HtmlContainer.cs#L383
Which you can get with same method from TappedEventArgs
https://github.com/AvaloniaUI/Avalonia/blob/fb0da85ad3a998bdf711b42e7b619550e6ff9de3/src/Avalonia.Base/Input/TappedEventArgs.cs#L20